### DISCUSSION TITLE
Como se coloca el boss de un usuario?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#347] [2. Questions and Answers] [answered]
Hola

He visto que en el web admin puedo colocar a un usuario, subordinate, pero estos subordinate no tiene relación con el boss.

**Ejemplo para un boss que le asigno usuarios en la base queda de esta manera**
```
"companies" : [
        {
            "hierarchy" : {
                "boss" : [

                ], 
                "peers" : [

                ], 
                "subordinate" : [
                    ObjectId("5e0a2cdb7e7bc5207a27088b")
                ]
            }, 
            "_id" : ObjectId("5e0a306eb490762011d850ae"), 
            "companyId" : ObjectId("5e0a1a1345e9602068206fc9")
        }
    ], 
```

**Sin embargo en la vista del subordinate no tiene el campo boss asignado.**

```
"companies" : [
        {
            "hierarchy" : {
                "boss" : [
                ], 
                "peers" : [

                ], 
                "subordinate" : [

                ]
            }, 
            "_id" : ObjectId("5e0a2cdb7e7bc5207a27088c"), 
            "companyId" : ObjectId("5e0a1a1345e9602068206fc9")
        }
    ], 
```

**Y si queremos invitar al jefe mediante del taskaddeditor, debo colocar el id en boss de la siguiente forma.**

```
"companies" : [
        {
            "hierarchy" : {
                "boss" : [
                    "5e0a306eb490762011d850ad"
                ], 
                "peers" : [

                ], 
                "subordinate" : [

                ]
            }, 
            "_id" : ObjectId("5e0a2cdb7e7bc5207a27088c"), 
            "companyId" : ObjectId("5e0a1a1345e9602068206fc9")
        }
    ], 
```

**Lo que se verá en el usuario desde el web admin al agregarlo desde la base de datos**

![image](https://user-images.githubusercontent.com/61419471/126841581-0342b8a7-1dee-4aaa-a4b7-74b5a402b0da.png)

Entonces mi pregunta es, cómo se debería colocar el boss al usuario sin base de datos: Es un feature o es un bug?.

Gracias =>

**Comment by guillevalin** [comment, marked as answer]
Gracias por tu feedback @NicolasPLG , efectivamente desde el administrador ahora sólo se pueden agregar subordinados, y en un futuro el selector de usuarios debería mostrar a través de este modelo de datos los jefes, solo que aún no ha sido actualizado.

Esperamos en un próximo release soportarlo de esta manera.
